src/components/SidebarPanel.js

```diff
@@ -0,0 +1,357 @@
+const initialState = {
+  isExpanded: false,
+  activeMediaPanel: null,
+  selectedTrack: null,
+  loading: false,
+  media: { audios: [], videos: [] },
+};
+
+// Reducer function
+function reducer(state, action) {
+  switch (action.type) {
+    case TOGGLE_SIDEBAR_ACTION:
+      return { ...state, isExpanded: action.payload };
+    case SET_ACTIVE_MEDIA_PANEL_ACTION:
+      return { ...state, activeMediaPanel: action.payload };
+    case SET_SELECTED_TRACK_ACTION:
+      return { ...state, selectedTrack: action.payload };
+    case CLOSE_MEDIA_PANEL_ACTION:
+      return { ...state, activeMediaPanel: null, selectedTrack: null };
+    case SET_LOADING_ACTION:
+      return { ...state, loading: action.payload };
+    case SET_MEDIA_ACTION:
+      return { ...state, media: action.payload };
+    case SET_ACTIVE_NOTES_PANEL_ACTION:
+      return { ...state, activeMediaPanel: null, selectedTrack: null };
+    case SET_ACTIVE_AUDIOS_PANEL_ACTION:
+      return { 
+        ...state, 
+        activeMediaPanel: 'audio', 
+        selectedTrack: state.media.audios[0] ?? null 
+      };
+    case SET_ACTIVE_VIDEOS_PANEL_ACTION:
+      return { 
+        ...state, 
+        activeMediaPanel: 'videos', 
+        selectedTrack: state.media.videos[0] ?? null 
+      };
+    default:
+      return state;
+  }
+}
+
+export default function SidebarPanel({ chapterTitle, locale, report, chapter, audios = [], videos = [] }) {
+  const [state, dispatch] = useReducer(reducer, { ...initialState, media: { audios, videos } });
+  const t = useTranslations('SidebarPanel');
+
+  const { isExpanded, activeMediaPanel, selectedTrack, loading, media } = state;
+
+  const menuItems = [
+    {
+      id: 'notes',
+      label: t('panel.notes') || 'Notes',
+      icon: FileText,
+      color: 'text-gray-700 hover:bg-blue-50',
+      items: ['Chapter Notes', 'My Notes', 'Bookmarks'],
+      onClick: () => dispatch({ type: SET_ACTIVE_NOTES_PANEL_ACTION }),
+    },
+    {
+      id: 'audio',
+      label: t('panel.audios') || 'Audio',
+      icon: Volume2,
+      color: 'text-gray-700 hover:bg-green-50',
+      onClick: () => dispatch({ type: SET_ACTIVE_AUDIOS_PANEL_ACTION }),
+      items: media.audios
+    },
+    {
+      id: 'videos',
+      label: t('panel.videos') || 'Videos',
+      icon: Video,
+      color: 'text-gray-700 hover:bg-red-50',
+      onClick: () => dispatch({ type: SET_ACTIVE_VIDEOS_PANEL_ACTION }),
+      items: media.videos
+    }
+  ];
+
+  return (
+    <>
+      {/* Fixed/Sticky Toggle Button - Only show when sidebar is collapsed */}
+      {!isExpanded && (
+        <button
+          onClick={() => dispatch({ type: TOGGLE_SIDEBAR_ACTION, payload: true })}
+          className="fixed top-20 left-4 z-50 w-12 h-12 bg-white border border-gray-200 shadow-lg flex items-center justify-center text-gray-600 hover:text-gray-800 hover:bg-gray-100 rounded-lg transition-all duration-200 hover:shadow-xl"
+          aria-label={t('expandSidebar') || 'Expand sidebar'}
```

Comment:
```suggestion
          aria-label={t('expandSidebar')}
```
The translation will never return null so the dead code is after `||` and apply for other translation as well.

Reply:
this is just for fallback in case the the translation is not configured

Reply:
when the translation is not configured, `t('expandSidebar')` return `expandSidebar` so `'Expand sidebar'` is a really deadcode :).